setup.py

```diff
@@ -17,25 +17,27 @@
 #
 # You should have received a copy of the GNU General Public License
 # along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+
 from setuptools import setup, find_packages
 import sys
 
+assert sys.version_info.major >= 3, "Python 3 is required to run PyNESTML"
+
```

Comment:
Failing on Python < 3 also makes out Python 2.7 build on TravisCI fail. That failure should be counted as a success.